tox.ini

```diff
@@ -21,6 +21,7 @@ passenv =
     dev: SENTRY_DSN
 deps =
     dev: ipython
+    dockercompose: docker-compose
```

Comment:
Apps use `requirements/dockercompose.txt` but packages don't have requirements files. So _for packages only_ if `"services"` is `"yes"` we need to add the `docker-compose` dependency to `tox.ini`